src/main/java/me/gaagjescraft/network/team/manhunt/managers/world/BukkitWorldManager.java

```diff
@@ -51,4 +51,11 @@ public void delete(String worldName) {
         }
     }
 
+    @Override
+    public void worldLoad(String worldName) {
+    }
+
+    @Override
+    public void worldUnload(String worldName){
```

Comment:
World unloading in Bukkit is also a thing, so you probably want to implement this as well.